src/frontends/pytorch/src/op/conv_transposend.cpp

```diff
@@ -0,0 +1,60 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset10.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_conv_transposend(NodeContext& context) {
+    auto strides = context.const_input<Strides>(3);
+    // In torch pads at beginning are same as at end
```

Comment:
please correct your comment to have it more clear

Reply:
pleasse check now it is enough clear?